contrib/windows_scripts/cubrid_env.bat

```diff
@@ -1,4 +1,20 @@
 @echo off
+
+REM  Copyright 2016 CUBRID Corporation
+REM
+REM   This program is free software; you can redistribute it and/or modify
+REM   it under the terms of the GNU General Public License as published by
+REM   the Free Software Foundation; either version 2 of the License, or
+REM   (at your option) any later version.
+REM
+REM  This program is distributed in the hope that it will be useful,
+REM  but WITHOUT ANY WARRANTY; without even the implied warranty of
+REM  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+REM  GNU General Public License for more details.
+REM
+REM  You should have received a copy of the GNU General Public License
+REM  along with this program; if not, write to the Free Software
+
```

Comment:
Please add one more line
`#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA`

Reply:
Thank you.
I will add it.